felt/src/lib.rs

```diff
@@ -781,6 +787,7 @@ mod test {
 
     proptest! {
```

Comment:
We have a problem there:
`proptest` does not run in `wasm`, as it uses fork processes, which is not possible in a `wasm` runtime.
So this run:
`cargo test --no-default-features`
But not this:
`wasm-pack test --node --no-default-features `

Therefore I'm pretty confident nothing is broken and everything is fine, but I still, tests cannot run.

Will it be possible to rework this part in order to get rid of proptest?

Reply:
Unless there's another framework that can run on wasm, the best I can offer is conditional compilation for proptest. Those are critical to QA and found real bugs several times already, so removing them for the general case is not a real option. I can research a bit, unless you already know of such a framework.

Reply:
Can't we conditionally run the proptests when not targeting wasm?